test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/local.settings.json

```diff
@@ -1,9 +1,9 @@
 {
-    "IsEncrypted": false,
-    "Values": {
-        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-        "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-        "OpenApi__Version":"v3",
-        "OpenApi__ForceHttps": "true"
-    }
-}
\ No newline at end of file
+  "IsEncrypted": false,

```

Comment:
Add `{` at the beginning.

Reply:
I don't understand well because of my short experience. I'm sorry, I just added the closing bracket in the end, but could you explain it again? :(

Reply:
umm.. should I do 'pull request' again?